src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -559,32 +559,35 @@ static int32_t ConsoleCommandGet(InteractiveConsole& console, const arguments_t&
         }
         else if (argv[0] == "park_value")
         {
-            console.WriteFormatLine("park_value %d", gameState.Park.Value / 10);
+            console.WriteLine(FormatString("park_value {CURRENCY2DP}", gameState.Park.Value));
         }
         else if (argv[0] == "company_value")
         {
-            console.WriteFormatLine("company_value %d", gameState.CompanyValue / 10);
+            console.WriteLine(FormatString("company_value {CURRENCY2DP}", gameState.CompanyValue));
         }
         else if (argv[0] == "money")
         {
-            console.WriteFormatLine("money %d.%d0", gameState.Cash / 10, gameState.Cash % 10);
+            console.WriteLine(FormatString("money {CURRENCY2DP}", gameState.Cash));
         }
         else if (argv[0] == "scenario_initial_cash")
         {
-            console.WriteFormatLine("scenario_initial_cash %d", gameState.InitialCash / 10);
+            console.WriteLine(FormatString("scenario_initial_cash {CURRENCY2DP}", gameState.InitialCash));
         }
         else if (argv[0] == "current_loan")
         {
-            console.WriteFormatLine("current_loan %d", gameState.BankLoan / 10);
+            console.WriteLine(FormatString("current_loan {CURRENCY2DP}", gameState.BankLoan));
         }
         else if (argv[0] == "max_loan")
         {
-            console.WriteFormatLine("max_loan %d", gameState.MaxBankLoan / 10);
+            console.WriteLine(FormatString("max_loan {CURRENCY2DP}", gameState.MaxBankLoan));
         }
         else if (argv[0] == "guest_initial_cash")
         {
-            console.WriteFormatLine(
-                "guest_initial_cash %d.%d0", gameState.GuestInitialCash / 10, gameState.GuestInitialCash % 10);
+            console.WriteLine(FormatString("guest_initial_cash {CURRENCY2DP}", gameState.GuestInitialCash));
+        }
+        else if (argv[0] == "construction_rights_cost")
```

Comment:
Why did you move this block up? It seems to me that keeping it together with "land price" makes more sense.

Reply:
Just to keep the currencies together which helped when comparing diffs. I can put it back if you want.

Reply:
Oh, just noticed I missed one anyway

Reply:
I moved both of them up.

Reply:
Could you put "land_rights_cost" above "construction_rights_cost"?  This is how they’re usually ordered. (Forgive me my high-octane autism here.)

Reply:
Done :)